libs/helm/sidebar/src/lib/hlm-sidebar.token.ts

```diff
@@ -0,0 +1,23 @@
+import { InjectionToken, ValueProvider, inject } from '@angular/core';
+
+export interface HlmSidebarConfig {
+	sidebarWidth: string;
+	sidebarWidthMobile: string;
+	sidebarWidthIcon: string;
+}
+
+const defaultConfig: HlmSidebarConfig = {
+	sidebarWidth: '16rem',
+	sidebarWidthMobile: '18rem',
+	sidebarWidthIcon: '4rem',
```

Comment:
shadcn/ui uses 3rem and the looks better with 3rem when using `collapsible` as icon

```suggestion
	sidebarWidthIcon: '3rem',
```